roles/pvc/mysql_pvc/files/mysql-persistent-template.yml

```diff
@@ -0,0 +1,137 @@
+---
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: mysql-persistent
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    template.openshift.io/expose-database_name: "{.data['database-name']}"
```

Comment:
What do the template.openshift.io annotations provide in this example?

I had assumed they'd be used for 'OpenShift Templates' but not seeing an OpenShift Template being used here.

Reply:
You are right, I'll remove it.